python/lsst/ts/ATDome/enums.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of ts_ATDome.
```

Comment:
Shouldn't this be rather placed as ATDome under ts_idl subsystem (python/lsst/ts/idl/enums path)?